modules/touch/src/touchmarker.cpp

```diff
@@ -79,6 +83,12 @@ TouchMarker::TouchMarker()
         glm::vec3(0.f),
         glm::vec3(1.f)
     )
+    , _inhouseColor(
+        InhouseColorInfo,
+        glm::vec3(51.f / 255.f, 51.f / 255.f, 204.f / 255.f),
+        glm::vec3(0.f),
+        glm::vec3(1.f)
+    )
```

Comment:
Maybe this should go in the header file

Reply:
Other files seems to specify vec3Property's in source. I cleared up the divisions to make it single-line.